cmd/docker/docker.go

```diff
@@ -170,17 +167,19 @@ func noArgs(cmd *cobra.Command, args []string) error {
 }
 
 func main() {
-	// Set terminal emulation based on platform as required.
-	stdin, stdout, stderr := term.StdStreams()
-	logrus.SetOutput(stderr)
+	logrus.SetOutput(os.Stderr)
```

Comment:
Are we happy that this is still the same thing even on Windows? I know that `term.StdStreams` will still be called implicitly by the following `NewDockerCli`. I think it's the same fd as you'd get here so maybe^Wprobably that's fine.

One other subtle change here I just realised is that `NewDockerCli` will now _always_ call `term.StdStreams()`, which unconditionally does some ioctls on stdio, even if the caller uses their own `WithStandardStreams` to set something else. I'm not going to pretend I know what those ioctls are doing -- are we sure they are safe and universally desirable?

Reply:
Good call. Can you change the default stdstreams handling in NewDockerCLI to be done at the end, only if there are missing streams (and if only stderr is missng, don't call term.StdStreams()).

Reply:
👍